docs/index.md

```diff
@@ -2214,11 +2214,16 @@ If no custom path was given, and if there are multiple configuration files in th
 1. `.mocharc.jsonc`
 1. `.mocharc.json`
 
+### Environment Variables
+
+The `MOCHA_OPTIONS` environment variable may be used to specify command line arguments. These arguments take priority over those found in configuration files.
```

Comment:
[Docs] I don't think we can expect all users to intuitively know how this setting CLI args through an env variable looks. How about a bit of explicit help like...

````suggestion
The `MOCHA_OPTIONS` environment variable may be used to specify command line arguments. These arguments take priority over those found in configuration files.

For example, setting the `bail` and `retries` options:

```bash
$ MOCHA_OPTIONS="--bail --retries 3" mocha
```
````

Reply:
Good call.